operator-rhel.yaml

```diff
@@ -50,7 +50,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
-              value: "hazelcast-operator"
+              value: "hazelcast-enterprise-operator"
             - name: RELATED_IMAGE_HAZELCAST
               value: registry.connect.redhat.com/hazelcast/hazelcast-4-rhel8:4.0-1
```

Comment:
Did we release 4.0.1? We can update that then.

Reply:
We did not, actually I am waiting for 4.0.1 MC image:
https://catalog.redhat.com/software/containers/detail/5e3d008b69aea31642954943